src/huggingface_hub/hf_api.py

```diff
@@ -405,6 +405,14 @@ def logout(self, token: Optional[str] = None) -> None:
         r = requests.post(path, headers={"authorization": "Bearer {}".format(token)})
         r.raise_for_status()
 
+    @staticmethod
+    def set_access_token(access_token: str):
+        write_to_credential_store("username", access_token)
```

Comment:
```suggestion
        write_to_credential_store("user", access_token)
```
I'd favor using user (i think we do this somewhere else)

Reply:
and maybe define a constant `USERNAME_PLACEHOLDER` to make intent clearer (and share it w/ the tests)